functions.php

```diff
@@ -113,9 +113,31 @@ function gesso_scripts() {
 	wp_register_script( 'gessoscripts', get_template_directory_uri() . '/js/dist/scripts.min.js', array( 'jquery', 'gessocommon', 'gessomodernizr' ), filemtime( get_template_directory() . '/js/dist/scripts.min.js' ) );
 	wp_enqueue_script( 'gessoscripts' );
 
-	// Add Google Fonts.
-	wp_register_style( 'gessoFonts', 'https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,300i,400,400i,600,600i,700,700i&display=swap', array(), $gesso_version );
-	wp_enqueue_style( 'gessoFonts' );
+		/**
+	* https://developer.wordpress.org/reference/hooks/style_loader_tag/
```

Comment:
@CodyPitts this should be places after the description with a leading `@see <link>` Docblock attribute.